server/peers.py

```diff
@@ -560,9 +560,11 @@ def retry_peer(self, peer, port_pairs):
             create_connection = self.proxy.create_connection
         else:
             create_connection = self.loop.create_connection
+            
+        laddr=(self.env.host, None) if self.env.host else None    
```

Comment:
Can you name this local_addr like the param and put spaces around the assignment operator on this line.  Then I'll merge.  Thanks.